lib/src/dap_resolver.dart

```diff
@@ -11,8 +11,18 @@ class DapResolver {
   http.Client client;
   DidResolver didResolver;
 
-  DapResolver(this.didResolver, {http.Client? client})
-      : client = client ?? http.Client();
+  static final _defaultMethodResolvers = [
+    DidJwkResolver(),
```

Comment:
no `DidJwk` becauuuse `did:jwk` doesn't support `service` in did docs

Reply:
gotcha!